pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -202,10 +211,38 @@ class CreateVM extends React.Component {
             break;
         }
 
+        let connectionUris = [
+            <Select.SelectEntry data={LIBVIRT_SYSTEM_CONNECTION}
+                                key={LIBVIRT_SYSTEM_CONNECTION}>{_("QEMU/KVM")}
```

Comment:
For symmetry and disambiguation, should this be labeled "QEMU/KVM System connection" then? And possibly "User session" → "User connection"? Might make sense to align this with how libvirt spells that in its documentation.

Reply:
I copied the naming from virt-manager UI. I think your suggestion sounds better.